src/cloudant/_common_util.py

```diff
@@ -124,19 +125,22 @@
     'group_sort': (STRTYPE, list),
     'include_docs': bool,
     'limit': (int, NONETYPE),
-    'query': (STRTYPE, int),
-    'q': (STRTYPE, int),
+    'query': (STRTYPE, int, LONGTYPE),
```

Comment:
In Py3 `LONGTYPE` is `NONETYPE`. We shouldn't allow NoneTypes to be passed as the query param.